seaborn/categorical.py

```diff
@@ -1230,24 +1230,49 @@ def estimate_statistic(self, estimator, ci, n_boot):
             self.value_label = "{}({})".format(estimator.__name__,
                                                self.value_label)
 
-    def draw_confints(self, ax, at_group, confint, colors, **kws):
+    def draw_confints(self,
+                      ax,
+                      at_group,
+                      confint,
+                      colors,
+                      draw_perp=False,
```

Comment:
IMO this parameter should be called `capsize` for consistency with matplotlib and should take a scalar value defaulting to 0. Then you can draw lines from `at - (capsize / 2)` to `at + (capsize / 2)`. Does that make sense?